modules/tinymce/src/core/main/ts/api/dom/DOMUtils.ts

```diff
@@ -1840,6 +1840,13 @@ const DOMUtils = (doc: Document, settings: Partial<DOMUtilsSettings> = {}): DOMU
     getContentEditable,
     getContentEditableParent,
 
+    /**
+     * Checks if the specified node is editable within the given context of it's parents.
+     *
+     * @method isEditable
+     * @param {Node} node Node to check if it's editable.
```

Comment:
```suggestion
     * @param {Node} Node to check if it's editable.
```

Reply:
I set the wrong review status after the previous comment ^^"

Reply:
The 'node' is the argument name so it should be there but it looks a bit confusing since the first word in the description is also node. So it's `@param {<type} <name> <description>`.

Reply:
[tsdoc](https://tsdoc.org/) suggests using a hyphen after the param name.

```ts
/**
   * Returns the average of two numbers.
   *
   * @remarks
   * This method is part of the {@link core-library#Statistics | Statistics subsystem}.
   *
   * @param x - The first input number
   * @param y - The second input number
   * @returns The arithmetic mean of `x` and `y`
   *
   * @beta
   */
```

Reply:
We are currently using a home grown doc generation tool. We had considered tsdoc in the past.